nekoyume/Assets/StreamingAssets/Localization/common.csv

```diff
@@ -1618,3 +1618,6 @@ UI_CHARGING_AP,Charging AP...,AP 충전중입니다.,Carregando AP ...,,充電AP
 UI_RECIPE_LOCK_GUIDE,"If you clear the stage more, the conditions will be revealed.",스테이지를 더 많이 클리어하면 조건이 공개됩니다.,"Se você limpar mais o estágio, as condições serão reveladas.",,もっと段階をクリアすると、条件が明らかにされます。,"Wenn Sie die Bühne mehr löschen, werden die Bedingungen offenbart.","Si aclara más el escenario, se revelarán las condiciones.",หากคุณล้างขั้นตอนเพิ่มเติมเงื่อนไขจะถูกเปิดเผย,"Jeśli usuniesz etap więcej, zostaną ujawnione warunki.","Jei išvalysite daugiau etapo daugiau, bus atskleistos sąlygos.","Als u het podium meer wist, worden de voorwaarden onthuld.","Jika Anda membersihkan tahap lebih, kondisinya akan terungkap.","Si vous effacez plus la scène, les conditions seront révélées.","Если вы очистите стадию больше, будут раскрыты условия.",ប្រសិនបើអ្នកឈូសឆាយដំណាក់កាលកាន់តែច្រើនលក្ខខណ្ឌនឹងត្រូវបានបង្ហាញឱ្យដឹង។,,如果您更清楚舞台，将会揭示条件。,
 UI_NOT_ENOUGH_HOURGLASS,Insufficient hourglass.,모래시계가 부족합니다.,,,,,,,,,,,,,,,,
 UI_RANKING_CATEGORY_ERROR,Error occured while loading category.,카테고리를 불러오는 도중 오류가 발생했습니다.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,
+UI_BLOCK_EQUIP,,장비를 장착할 수 없습니다.,,,,,,,,,,,,,,,,
+UI_BLOCK_UNEQUIP,,장비를 해제할 수 없습니다.,,,,,,,,,,,,,,,,
+UI_BLOCK_CHARGE_AP,,AP를 충전할 수 없습니다.,,,,,,,,,,,,,,,,
```

Comment:
I think you should fill english label in order to work fallback localization well.

Reply:
I asked to @Namyujeong for other language. and I'm going to fill it. Can this be okay?